test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -9578,6 +9579,79 @@ def create_import_hook(self, name, attrs=None, body=None, overwrite=True,
 
         return ret
 
+    def import_hook_config(self, hook_name, hook_conf, hook_type,
+                           level=logging.INFO):
+        """
+        Helper function to import hook config body into hook by name.
+        The hook must have been created prior to calling this
+        function.
+
+        :param hook_name: The name of the hook to import body to
+        :type name: str
+        :param hook_conf: The body of the hook config as a string.
+        :type hook_conf: str
```

Comment:
shouldn't this be dict instead str?

Reply:
User should pass dict quoted as string so that we will write it into a file and export

Reply:
I think it will be better to allow use pass dict and this method will use `json.dumps` then write in the file.
Think of this use case: user calls export_hook_conf then updates returned dict then calls this method. it will be simple for the user else the user has to convert dict into a string then pass it to this.

Reply:
i agree with @hirenvadalia here, the user need not do the conversion.